markup/house-style.md

```diff
@@ -75,6 +76,55 @@ We *don't* do this:
 ```html
 <meta charset="UTF-8" />
 ```
+### Form submit controls
+
+On modern browsers, `button type="submit"` provides a superset of functionality when compared to `input type=submit`.
+
+`button` elements are much easier to style than `input` elements. You can add
+inner HTML content (think `<em>`, `<strong>`, or even `<img>`), and use
+`::after` and `::before` pseudo-elements to achieve complex rendering while
+`input` only accepts a textual value attribute.
+
+Make sure to specify `name` and `value` attributes to maximize browsers
+compatibility.
+
+We do this:
+
+```html
+<form>
+	<button type="submit" name="myButton" value="foo">Continue</button>
+</form>
+```
+
+or
+
+We *don't* do this:
+
+```html
+<form>
+	<input type="submit" value="Continue">
+</form>
+```
+
```

Comment:
Should we also add this (submitting is the default behaviour, but we want to "code for humans")?

We do this:

```html
<form>
	<button type="submit" …
</form>
```

We *don't* do this:

```html
<form>
	<button …
</form>
```